payment-flow-service-api/src/main/java/com/aevi/sdk/pos/flow/stage/CardReadingModel.java

```diff
@@ -91,23 +90,31 @@ public TransactionRequest getTransactionRequest() {
     /**
      * Approve the card reading with a valid card instance.
```

Comment:
Minor but probably worth mentioning this sends it straight away as well

Reply:
yeap fair point